From 2b68b67797436389b21cd28242051ccaec52582f Mon Sep 17 00:00:00 2001
From: Bjoern Michaelsen <bjoern.michaelsen@canonical.com>
Date: Wed, 22 May 2013 07:22:33 +0200
Subject: [PATCH] fdo#63840 lp#1182082: disable internal unity integration
 against glib<2.36

* that is: use internal unity integration from Ubuntu 13.04 raring
  upwards only

Change-Id: I08b9d950245b21e8d92c1e76905d53e97afdf685
---
 vcl/inc/unx/gtk/gtksalmenu.hxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vcl/inc/unx/gtk/gtksalmenu.hxx b/vcl/inc/unx/gtk/gtksalmenu.hxx
index fc0632f..415aa4e 100644
--- a/vcl/inc/unx/gtk/gtksalmenu.hxx
+++ b/vcl/inc/unx/gtk/gtksalmenu.hxx
@@ -19,7 +19,7 @@
 #include <unx/gtk/gtkframe.hxx>
 
 #if defined(ENABLE_DBUS) && defined(ENABLE_GIO) && \
-    (GLIB_MAJOR_VERSION > 2 || GLIB_MINOR_VERSION >= 32)
+    (GLIB_MAJOR_VERSION > 2 || GLIB_MINOR_VERSION >= 36)
 #  define ENABLE_GMENU_INTEGRATION
 #  include <unx/gtk/glomenu.h>
 #  include <unx/gtk/gloactiongroup.h>
-- 
1.8.3

